app/ducks/walletActions.js

```diff
@@ -332,7 +332,9 @@ async function parseInputsOutputs(net, tx) {
 
     // Track normal receive amounts for later
     if (covenant.action === 'NONE') {
-      totalValue += output.value;
+      if (await walletClient.hasAddress(output.address)) {
```

Comment:
Isn't this the same as just checking if `output.path` is non-null?

Reply:
didn't know there is a `output.path` variable 😄  

fixed now! I also fix `wallet/service` to inflate transactions before returning.